### DISCUSSION TITLE
channel name   How to configure the name to support Chinese

### DISCUSSION THREAD

**Discussion by ourlang** [nextgenhealthcare/connect#6244] [Q&A] [answered]
channel name   How to configure the name to support Chinese
![image](https://github.com/nextgenhealthcare/connect/assets/32355185/2558ab88-a921-4901-b85e-ec9adf851741)
Entering Chinese is invalid for channel name

**Comment by pacmano1** [comment, marked as answer]
Not possible. See https://github.com/nextgenhealthcare/connect/issues/5570 and perhaps add your feedback.
